diff -aurN mksh.orig/check.t mksh/check.t
--- mksh.orig/check.t	2014-01-14 18:44:49.083485090 +0100
+++ mksh/check.t	2014-01-14 20:39:23.502240931 +0100
@@ -6766,7 +6766,6 @@
 	nohup='nohup '
 	r='fc -e -'
 	source='PATH=$PATH:. command .'
-	stop='kill -STOP'
 	type='whence -v'
 ---
 name: aliases-1-hartz4
@@ -6834,7 +6833,6 @@
 	nohup='nohup '
 	r='fc -e -'
 	source='PATH=$PATH:. command .'
-	stop='kill -STOP'
 	type='whence -v'
 ---
 name: aliases-3b
@@ -6857,7 +6855,6 @@
 	nohup='nohup '
 	r='fc -e -'
 	source='PATH=$PATH:. command .'
-	stop='kill -STOP'
 	type='whence -v'
 ---
 name: aliases-2b-hartz4
diff -aurN mksh.orig/main.c mksh/main.c
--- mksh.orig/main.c	2014-01-14 18:44:49.082485051 +0100
+++ mksh/main.c	2014-01-14 20:40:00.826678427 +0100
@@ -71,7 +71,7 @@
 	/* not "alias -t --": hash -r needs to work */
 	"hash=alias -t",
 	"type=whence -v",
-#if !defined(ANDROID) && !defined(MKSH_UNEMPLOYED)
+#if 0
 	/* not in Android for political reasons */
 	/* not in ARGE mksh due to no job control */
 	"stop=kill -STOP",
diff -aurN mksh.orig/mksh.1 mksh/mksh.1
--- mksh.orig/mksh.1	2014-01-14 18:44:49.082485051 +0100
+++ mksh/mksh.1	2014-01-14 20:40:10.388046659 +0100
@@ -1131,7 +1131,6 @@
 nameref=\*(aqtypeset \-n\*(aq
 nohup=\*(aqnohup \*(aq
 r=\*(aqfc \-e \-\*(aq
-stop=\*(aqkill \-STOP\*(aq
 type=\*(aqwhence \-v\*(aq
 .Ed
 .Pp
